data/json/npcs/TALK_CYBORG_1.json

```
@@ -2,10 +2,150 @@
+        "success": { "topic": "TALK_CYBORG_FRIENDLY", "opinion": { "trust": 1, "value": 1 } },
+        "failure": { "topic": "TALK_CYBORG_LEAVES" }
+      },
+      {
+        "text": "For all you know, I did. I'm being nice for now. You'd better hope that it lasts.",
+        "trial": { "type": "INTIMIDATE", "difficulty": 20, "mod": [ [ "AGGRESSION", -2 ], [ "BRAVERY", -2 ] ] },
+        "success": { "topic": "TALK_CYBORG_FEARFUL", "opinion": { "trust": -4, "fear": 3, "value": -1, "anger": 4 } },
+        "failure": { "topic": "TALK_CYBORG_BREAKDOWN", "opinion": { "trust": -4, "value": -5, "anger": 10 } }
+      },
+      {
+        "text": "Forget it. Enjoy your freedom.",
+        "topic": "TALK_DONE"
+      }
+    ]
+  },
+  {
+    "id": "TALK_CYBORG_FEARFUL",
+	"type": "talk_topic",
+	"dynamic_line": "Okay, okay, *BUZZ* I'm sorry! Don't hurt me again! Anything but the chip!",
+	"responses": [
+      {
+        "text": "Follow me and do my bidding, then.",
+        "trial": { "type": "INTIMIDATE", "difficulty": 20, "mod": [ [ "FEAR", 8 ], [ "VALUE", 2 ], [ "TRUST", 2 ], [ "BRAVERY", -2 ] ] },
+        "success": { "topic": "TALK_AGREE_FOLLOW", "effect": "follow", "opinion": { "trust": -4, "fear": 3, "value": -1, "anger": 4 } },
+        "failure": { "topic": "TALK_CYBORG_BREAKDOWN", "opinion": { "trust": 4, "value": -5, "anger": 10 } }
+      },
+      {
+        "text": "Get out of my sight.",
+        "topic": "TALK_DONE",
+        "effect": "flee",
+        "opinion": { "trust": -1, "fear": -2, "anger": -2 }
+      },
+      {
+        "text": "No, *I'm* sorry, I didn't mean that. Go do what you want.",
+        "trial": { "type": "PERSUADE", "difficulty": 10 },
+        "success": { "topic": "TALK_DONE", "opinion": { "trust": 1, "value": 1, "anger": -1 } },
+        "failure": { "topic": "TALK_CYBORG_BREAKDOWN", "opinion": { "trust": 4, "value": -5, "anger": 10 } }
+      }
+	]
+  },
+  {
+    "id": "TALK_CYBORG_BREAKDOWN",
+	"type": "talk_topic",
+	"dynamic_line": "...kill... *ZTZTZT* ...you!",
+	"responses": [
+      {
+        "text": "Run while you still can!",
+        "trial": { "type": "INTIMIDATE", "difficulty": 20, "mod": [ [ "FEAR", 8 ], [ "VALUE", 2 ], [ "TRUST", 2 ], [ "BRAVERY", -2 ] ] },
+        "success": { "topic": "TALK_DONE", "effect": "flee" },
+        "failure": { "topic": "TALK_DONE", "effect": "hostile" }
+      },
+      {
+        "text": "Not if I kill you first!",
+        "topic": "TALK_DONE",
+        "effect": "hostile"
+      },
+      {
+        "text": "I'm sorry! I shouldn't have said that!",
+        "trial": { "type": "PERSUADE", "difficulty": 30 },
+        "success": { "topic": "TALK_DONE", "opinion": { "trust": 1, "value": 1, "anger": -1 } },
+        "failure": { "topic": "TALK_DONE", "effect": "hostile" }
+      }
+	]
+  },
+  {
+	"id": "TALK_CYBORG_LEAVES",
+	"type": "talk_topic",
+	"dynamic_line": "Yeah, *beeeeep* I don't think so. Bye.",
+	"responses": [
```

```suggestion
	"responses": [ { "text": "Suit yourself.", "topic": "TALK_DONE" } ]
```
Again, the linter must be appeased.